glean_parser/parser.py

```diff
@@ -253,6 +256,7 @@ def _instantiate_metrics(
                         f"Duplicate metric name '{category_key}.{metric_key}' "
                         f"already defined in '{already_seen}'"
                     ),
+                    metric_val.defined_in["line"],
```

Comment:
`metric_val` is not defined here, but you should be able to use `metric_obj` instead.

```suggestion
                    metric_obj.defined_in["line"],
```